src/style/components/list/participant-item.less

```diff
@@ -109,10 +109,10 @@
         display: flex;
         min-width: 0; // this will ensure that ellipses is working
         height: @avatar-diameter-m;
-        flex-direction: column;
         flex-grow: 1;
-        align-items: flex-start;
-        justify-content: center;
+        align-items: center;
+        justify-content: flex-start;
+        font-size: @font-size-medium!important;
```

Comment:
is there anyway to avoid the important on this? where is it inheriting from?

Reply:
good catch, forgot about that one.
It's inheriting it from the text-medium class that I'm hesitant to change even it seems like it's outdated